invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Home/components/lists/OverdueLoansList/__tests__/OverdueLoansList.js

```diff
@@ -105,17 +108,20 @@ describe('OverdueLoansList tests', () => {
       push: mockedHistoryPush,
     };
 
-    const data = [
-      {
-        ID: 'loan1',
-        loan_pid: 'loan1',
-        patron_pid: 'patron_1',
-        updated: d,
-        created: d,
-        start_date: d,
-        end_date: d,
-      },
-    ];
+    const data = {
+      hits: [
+        {
+          ID: 'loan1',
```

Comment:
Is this `ID` required? I haven't seen it in any of the previous test data. The same question stands for all the occurrences further down.